app/src/main/java/pl/futurecollars/invoicing/model/Invoice.java

```diff
@@ -0,0 +1,26 @@
+package pl.futurecollars.invoicing.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+@Data
+public class Invoice {
+
+    private UUID id = UUID.randomUUID();
```

Comment:
It is better to initialize it in constructor. You can override empty constructor and set value to this variable.